staging/src/k8s.io/cloud-provider/cloud.go

```diff
@@ -104,6 +104,12 @@ func GetInstanceProviderID(ctx context.Context, cloud Interface, nodeName types.
 }
 
 // LoadBalancer is an abstract, pluggable interface for load balancers.
+// PR https://github.com/kubernetes/kubernetes/pull/80660 added support for cloud providers to return an error
```

Comment:
```
LoadBalancer is an abstract, pluggable interface for load balancers.

Cloud provider may chose to implement the logic for
constructing/destroying specific kinds of load balancers in a
controller separate from the ServiceController.  If this is the case,
then {Ensure,Update}LoadBalancer and EnsureLoadBalancerDeleted must
return the ImplementedElsewhere error.  For the given LB, the
GetLoadBalancer must return NotFound and GetLoadBalancerName() return
empty string. EnsureLoadBalancerDeleted must not return
ImplementedElsewhere to ensure proper teardown of resources that
allocated by the ServiceController. (This can happen if a user changes
the type of LB via an update to the resource.)
```